bpfd-operator/controllers/bpfd-agent/discovered_program.go

```diff
@@ -68,7 +68,12 @@ func (r *DiscoveredProgramReconciler) SetupWithManager(mgr ctrl.Manager) error {
 
 // Reconcile ALL discovered bpf programs on the system whenever an event is received.
 func (r *DiscoveredProgramReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.Result, error) {
-	r.Logger = log.FromContext(ctx)
+	r.Logger = ctrl.Log.WithName("discvr")
+
+	// The Discovery Reconciler gets called more than others so this tends to be a noisy log,
+	// so moved to a Trace log.
+	ctxLogger := log.FromContext(ctx)
+	ctxLogger.V(2).Info("Reconcile DiscoveredProgs: Enter", "req", req)
```

Comment:
`ctxLogger.V(2)` what's the log level that needs to be configured to see this? I assume it's past `debug`

Reply:
Yes, good catch. I thought it was "trace" but forgot to test it. bpfd-agent/main.go and bpfd-operator/main.go only process "info" and "debug" The https://github.com/go-logr/zapr does have a level, I will play with that. If that doesn't work, not sure what to do with noisy logs since we are defaulting to "debug".

Reply:
Yeah feel free to map a a level to an int if needed